Assets/MRTK/SDK/Features/UX/Scripts/Utilities/FeaturesPanelVisuals.cs

```diff
@@ -27,8 +27,15 @@ void Start()
         {
             profilerButton.IsToggled = (CoreServices.DiagnosticsSystem?.ShowProfiler).GetValueOrDefault(false);
             handRayButton.IsToggled = PointerUtils.GetPointerBehavior<ShellHandRayPointer>(Handedness.Any, InputSourceType.Hand) != PointerBehavior.AlwaysOff;
-            handMeshButton.IsToggled = (CoreServices.InputSystem?.InputSystemProfile?.HandTrackingProfile.EnableHandMeshVisualization).GetValueOrDefault(false);
-            handJointsButton.IsToggled = (CoreServices.InputSystem?.InputSystemProfile?.HandTrackingProfile.EnableHandJointVisualization).GetValueOrDefault(false);
+
+            MixedRealityHandTrackingProfile handProfile = null;
+            if ((CoreServices.InputSystem != null) &&
+                (CoreServices.InputSystem.InputSystemProfile != null))
```

Comment:
You should still be able to use null propagation on `CoreServices.InputSystem` if you wanted to simplify, but either way works!

```suggestion
            if (CoreServices.InputSystem?.InputSystemProfile != null)
```